app/Http/Controllers/Secretariat/UserController.php

```diff
@@ -302,7 +302,7 @@ public function export()
     {
         $this->authorize('viewAny', User::class);
 
-        return Excel::download(new UsersExport(), 'uran_export.xlsx');
+        return Excel::download(new UsersExport(User::canView()), 'uran_export.xlsx');
```

Comment:
You don't use this controller function anymore. Delete it together with it's endpoint.

Reply:
I think it's still in use: it's referenced in `evaluation_form_available_details.blade.php` and in `evaluation_form_closed.blade.php`. (Look for `<a href="{{ route('users.export') }}">letölthetőek</a>` in both files.)

Reply:
Hm, true but it might create some confusion in the future. These are only in emails, we could change the link to point to the users index page.

Reply:
Done